index.html

```diff
@@ -56,6 +57,136 @@ <h1 class="describe">
       </button>
     </div>
   </section>
+  <section class="achieved-work">
+    <h1 class="section-title">My Recent Work</h1>
```

Comment:
- [ ] To keep the HTML semantic, you should consider changing this to a `<h2>` tag instead. Even though using more than one `<h1>` is allowed by the HTML specification, it is not considered a best practice, you can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).